packages/gcp/src/__tests__/ServiceWrapper.test.ts

```diff
@@ -3,33 +3,36 @@
  */
 
 import { ProjectsClient } from '@google-cloud/resource-manager'
-import { compute_v1 } from 'googleapis'
-import { compute as googleCompute } from 'googleapis/build/src/apis/compute'
+import {
+  AddressesClient,
+  DisksClient,
+  ImagesClient,
+  InstancesClient,
+  MachineTypesClient,
+  protos as googleCompute,
+} from '@google-cloud/compute'
 import { GoogleAuth } from 'google-auth-library'
 import { RecommenderClient } from '@google-cloud/recommender'
 import { GoogleAuthClient, wait } from '@cloud-carbon-footprint/common'
 import {
-  InstanceData,
-  mockedAddressesResultItems,
+  mockAddressesResultItems,
   mockedAddressGetDetails,
   mockedDisksGetSSDDetails,
-  mockedDisksResultItems,
+  mockDisksResultItems,
```

Comment:
nitpick - this change makes sense, but diverges from the other naming conventions consistent with other 'mocked' data imports

Reply:
The `mock_ResultItems` names were all change because they are now all functions to generate a new AsyncIterator containing the mock data instead of the flat mock data object that it was before. I changed all of the names that use these functions to `mock` to make it sound more actionable so that it's clear it's a function. Happy for suggestions on naming though.